Example/Podfile

```diff
@@ -1,5 +1,3 @@
-use_frameworks!
```

Comment:
just out of curiosity: why this thing is removed?

Reply:
React Native projects don't have that by default, so i thought lack of `use_frameworks` is closer to the default use case scenario. 
also, to be clear: that's just Example app's configuration, it doesn't have any effect on the actual `DatadogSDKBridge`